jupyter/datascience/anaconda-python-3.8/requirements.in

```diff
@@ -0,0 +1,28 @@
+# Using this to generate exact versions in environment.yml
+
```

Comment:
Use Pipefile instead requirements.in

Reply:
This doesn't use pip, I just am using that to generate the exact versions of packages in the environment.yml